api/src/main/java/org/openmrs/module/cmrreports/Item.java

```diff
@@ -24,9 +24,10 @@
 /**
  * Please note that a corresponding table schema must be created in liquibase.xml.
```

Comment:
@rubailly In future please commit formatting changes and code changes sepparately :)